modules/credentials/variables.tf

```diff
@@ -3,3 +3,9 @@ variable "resource_prefix" {
   default     = ""
   description = "An optional prefix to add to all resources created."
 }
+
+variable "permissions_boundary_arn" {
+  type        = string
+  default     = null
```

Comment:
I'm not sure that type `string` can be `null`. Might need to be `""`

Reply:
Oh yeah I meant to change that back, good call